builtin/game/chat.lua

```diff
@@ -742,13 +742,17 @@ core.register_chatcommand("mods", {
 	description = S("List mods installed on the server"),
 	privs = {},
 	func = function(name, param)
+		core.show_formspec(name, "__builtin:mods", build_mods_formspec())
+		return true
+	end
+	--[[func = function(name, param)
```

Comment:
Why are you adding a breaking change here instead of adding a new `/mods_gui` command?

Does this change the usage of `/mods` for scripting or dedicated servers?

WHy do you comment out code instead of removing it?

Reply:
`/mods` command now always outputs only the gui, not the string as before. And it doesn't make sense to me adding still the same command (`/mods_gui`), what for? And yes, the commented code I will remove.